style/ruby/README.md

```diff
@@ -0,0 +1,14 @@
+# Ruby Style Enforcement
+
+It is recommended that you use a ruby style enforcement tool such as [RuboCop](/bbatsov/rubocop). This can be integrated in your CI build and optionally use a service like HoundCI to comment on Pull Requests indicating violations.
+
+There is a RuboCop configuration file available in this directory that can be used as a base for a new project.  There are various modifications in that configuration that we have collaboratively determined are acceptable in DLSS.
```

Comment:
The thoughtbot one is short and nice, and I'm :+1: to referencing it.. I'm still looking for something that captures my ideas about the "toolness" of code linting, but maybe "A reason is good when you can convince a teammate." gets to the same result.